swssconfig/swssconfig.cpp

```diff
@@ -127,47 +133,78 @@ bool load_json_db_data(iostream &fs, vector<KeyOpFieldsValuesTuple> &db_items)
     return true;
 }
 
+vector<string> read_directory(const string &path)
+{
+    vector<string> ret;
+    DIR *dir;
+    dirent *entry;
+    dir = opendir(path.c_str());
+    if (dir)
+    {
+        while (true)
+        {
+            entry = readdir(dir);
+            if (entry == NULL) break;
+            if (strcmp(entry->d_name, ".") && strcmp(entry->d_name, ".."))
```

Comment:
formatting ,p lease add { } on every if, and don't inline code like that, 1 statement per line